README.md

```diff
@@ -403,15 +403,14 @@ function example(string $name) {
 
 The most commonly used tags for Strings are:
```

Comment:
Maybe get rid of this line? Take out the grey area, these are the tags you can use. What do you think?

Reply:
Seems legit to me! On it.